database/service_db.go

```diff
@@ -77,39 +101,76 @@ func (d *LevelDBServiceDB) All() ([]*service.Service, error) {
 }
 
 // Delete deletes service from database.
-func (d *LevelDBServiceDB) Delete(id string) error {
-	return d.db.Delete([]byte(id), nil)
+// TODO this should also find and delete alias when id is given.
+func (d *LevelDBServiceDB) Delete(idOrAlias string) error {
```

Comment:
this should also find and delete alias when id is given

Reply:
done